test/WorkflowCore.Tests.SqlServer/SqlServerQueueProviderFixture.cs

```diff
@@ -0,0 +1,140 @@
+﻿#region using
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using FluentAssertions;
+
+using WorkflowCore.Interface;
+using WorkflowCore.QueueProviders.SqlServer.Services;
+
+using Xunit;
+using Xunit.Abstractions;
+
+#endregion
+
+namespace WorkflowCore.Tests.SqlServer
+{
+    [Collection("SqlServerBroker collection")]
+    public class SqlServerQueueProviderFixture : IDisposable
+    {
+        #region Init
+
+        readonly SqlServerQueueProvider _qb;
+        private readonly ITestOutputHelper _console;
+
+        public SqlServerQueueProviderFixture(ITestOutputHelper output)
```

Comment:
Maybe it makes sense to build a suite of tests that all queue providers can inherit from but just implement there own setup and tear down?